web/server/README.md

```diff
@@ -33,7 +33,7 @@ The ports to bind are controlled via `[web].bind to`, like this:
 ```
 [web]
    default port = 19999
-   bind to = 127.0.0.1=dashboard^SSL=optional 10.1.1.1:19998=management|netdata.conf hostname:19997=badges [::]:19996=streaming^SSL=force localhost:19995=registry *:http=dashboard unix:/tmp/netdata.sock
+   bind to = 127.0.0.1=dashboard^SSL=optional 10.1.1.1:19998=management|netdata.conf hostname:19997=badges [::]:19996=streaming^SSL=force localhost:19995=registry *:http=dashboard unix:/run/netdata/netdata.sock
```

Comment:
Please, let us use as suggestion `/var/run`, because `/run` is present only on Linux as you can see in this screenshot from FreeBSD:

![VirtualBox_FreeBSD_27_10_2019_19_37_34](https://user-images.githubusercontent.com/49162938/67640250-5813a500-f8f1-11e9-9b20-aada42629f4a.png)